apps/tasks/lib/main.dart

```diff
@@ -26,8 +26,8 @@ class MyApp extends StatelessWidget {
           create: (_) => LanguageModel(),
         ),
       ],
-      child: Consumer2<ThemeModel, LanguageModel>(
-          builder: (_, ThemeModel themeNotifier, LanguageModel languageNotifier, __) {
+      child: Consumer2<ThemeModel, LanguageModel>(builder:
```

Comment:
Have you changed your formatter settings?

Reply:
No, but probably they aren't the same across different devices. Maybe we should make a team decision about line wrapping length for "normal cases".